core/src/banking_stage/worker.rs

```diff
@@ -0,0 +1,608 @@
+use {
+    super::{
+        consumer::Consumer,
+        forwarder::Forwarder,
+        scheduler_messages::{ConsumeWork, FinishedConsumeWork, FinishedForwardWork, ForwardWork},
+        ForwardOption,
+    },
+    crossbeam_channel::{select, Receiver, RecvError, SendError, Sender},
+    solana_poh::leader_bank_notifier::LeaderBankNotifier,
+    solana_runtime::bank::Bank,
+    std::{sync::Arc, time::Duration},
+    thiserror::Error,
+};
+
+#[derive(Debug, Error)]
+pub enum WorkerError {
+    #[error("Failed to receive work from scheduler: {0}")]
+    Recv(RecvError),
```

Comment:
nit try `Recv(#[from] RecvError),`

Reply:
(friendly bump)

Reply:
Ha was waiting for my #31542 to get in.

done: a6ac9f65971ca8ab0bdc75649ae5e14dea4dbceb